From 87f454532a612066c3caacd240782fc40f31c152 Mon Sep 17 00:00:00 2001
From: Oliver Steffen <osteffen@redhat.com>
Date: Tue, 7 Jan 2025 12:58:21 +0100
Subject: [PATCH] SecurityPkg/Tpm2DeviceLibDTpm: Improve spelling/grammar of
 comments

Fix some spelling/grammar mistakes in the documentation comments.

Suggested-by: Dionna Glaze <dionnaglaze@google.com>
Signed-off-by: Oliver Steffen <osteffen@redhat.com>
---
 .../Library/Tpm2DeviceLibDTpm/Tpm2DeviceLibDTpm.c  | 14 +++++++-------
 .../Tpm2DeviceLibDTpm/Tpm2InstanceLibDTpm.c        |  4 ++--
 SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2Ptp.c    |  2 +-
 3 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2DeviceLibDTpm.c b/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2DeviceLibDTpm.c
index 7cc55df436..e0897417ba 100644
--- a/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2DeviceLibDTpm.c
+++ b/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2DeviceLibDTpm.c
@@ -79,9 +79,9 @@ Tpm2SubmitCommand (
 }
 
 /**
-  This service requests use TPM2.
+  This service requests to use TPM2.
 
-  @retval EFI_SUCCESS      Get the control of TPM2 chip.
+  @retval EFI_SUCCESS      Get the control of the TPM2 chip.
   @retval EFI_NOT_FOUND    TPM2 not found.
   @retval EFI_DEVICE_ERROR Unexpected device behavior.
 **/
@@ -95,13 +95,13 @@ Tpm2RequestUseTpm (
 }
 
 /**
-  This service register TPM2 device.
+  This service registers a TPM2 device.
 
   @param Tpm2Device  TPM2 device
 
-  @retval EFI_SUCCESS          This TPM2 device is registered successfully.
-  @retval EFI_UNSUPPORTED      System does not support register this TPM2 device.
-  @retval EFI_ALREADY_STARTED  System already register this TPM2 device.
+  @retval EFI_SUCCESS          TPM2 device was registered successfully.
+  @retval EFI_UNSUPPORTED      System does not support registering this TPM2 device.
+  @retval EFI_ALREADY_STARTED  This TPM2 device is already registered.
 **/
 EFI_STATUS
 EFIAPI
@@ -115,7 +115,7 @@ Tpm2RegisterTpm2DeviceLib (
 /**
   The function caches current active TPM interface type.
 
-  @retval EFI_SUCCESS   DTPM2.0 instance is registered, or system does not support register DTPM2.0 instance
+  @retval EFI_SUCCESS   DTPM2.0 instance is registered, or system does not support registering a DTPM2.0 instance
 **/
 EFI_STATUS
 EFIAPI
diff --git a/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2InstanceLibDTpm.c b/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2InstanceLibDTpm.c
index 7d3e4bef86..11796d5b97 100644
--- a/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2InstanceLibDTpm.c
+++ b/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2InstanceLibDTpm.c
@@ -69,9 +69,9 @@ TPM2_DEVICE_INTERFACE  mDTpm2InternalTpm2Device = {
 };
 
 /**
-  The function register DTPM2.0 instance and caches current active TPM interface type.
+  Registers DTPM2.0 instance and caches current active TPM interface type.
 
-  @retval EFI_SUCCESS   DTPM2.0 instance is registered, or system does not support register DTPM2.0 instance
+  @retval EFI_SUCCESS   DTPM2.0 instance is registered, or system does not support registering a DTPM2.0 instance
 **/
 EFI_STATUS
 EFIAPI
diff --git a/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2Ptp.c b/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2Ptp.c
index eac9f0e299..dee01d9707 100644
--- a/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2Ptp.c
+++ b/SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2Ptp.c
@@ -1,5 +1,5 @@
 /** @file
-  PTP (Platform TPM Profile) CRB (Command Response Buffer) interface used by dTPM2.0 library.
+  PTP (Platform TPM Profile) CRB (Command Response Buffer) interface used by DTPM2.0 library.
 
 Copyright (c) 2015 - 2021, Intel Corporation. All rights reserved.<BR>
 Copyright (c), Microsoft Corporation.
-- 
2.43.0

